galsim/bandpass.py

```diff
@@ -229,15 +229,15 @@ def __mul__(self, other):
         if hasattr(other, '__call__'):
             tp = lambda w: self.func(w) * other(w)
         elif isinstance(self._tp, galsim.LookupTable):
-            # If other is not a function, then there is no loss of accuracy by applying the 
+            # If other is not a function, then there is no loss of accuracy by applying the
             # factor directly to the LookupTable, if that's what we are using.
             # Make sure to keep the same properties about the table, wave_type.
             if self.wave_factor == 10.0:
                 wave_type = 'Angstroms'
             x = self._tp.getArgs()
-            f = [ val * other for val in self._tp.getVals() ]
+            f = [val * other for val in self._tp.getVals()]
```

Comment:
Here's another one.  I don't understand why pep8 doesn't like spaces at the start and end of lists.  It makes them much easier to read with the initial space after the `[` or `(` or `{`.